tests/js/spec/components/events/eventCauseEmpty.spec.jsx

```diff
@@ -60,7 +60,7 @@ describe('EventCauseEmpty', function() {
     wrapper.update();
 
     wrapper
-      .find('[data-test-id="snoozed"]')
+      .find('button[data-test-id="snoozed"]')
```

Comment:
This is old code, but can we test with an `aria-label` selector, e.g. `button[aria-label="Snoozed"]`

Our buttons will automatically add an `aria-label` and we can avoid the extra `data-test-id` attribute. In the case where we use an icon for a button, ideally we'd add a `aria-label`.

As a side effect, we can get in the practice of having our buttons be more accessible.